xcdat/regridder/xesmf.py

```diff
@@ -77,6 +80,12 @@ def __init__(
             regridding methods.
         unmapped_to_nan : bool
             Sets values of unmapped points to `np.nan` instead of 0 (ESMF default).
+        output_weights : bool | str
+            If True, output weights are added to the output dataset as weights.
+            If str, the name of the variable to store the weights. Default is False.
+        create_nan_mask : bool
+            If True, a mask is created using the nan values from source variable. If
+            a mask already exists in the Dataset it will be ignored.
```

Comment:
```suggestion
        create_nan_mask : bool
            If True, a mask is created using the nan values from source variable. If
            a mask already exists in the Dataset it will be ignored. Default is False.
```